lib/rex/proto/kerberos/model/error.rb

```diff
@@ -108,7 +112,20 @@ module ErrorCodes
             KRB_AP_ERR_INAPP_CKSUM = ErrorCode.new('KRB_AP_ERR_INAPP_CKSUM', 50, 'Inappropriate type of checksum in message')
             KRB_ERR_GENERIC = ErrorCode.new('KRB_ERR_GENERIC', 60, 'Generic error')
             KRB_ERR_FIELD_TOOLONG = ErrorCode.new('KRB_ERR_FIELD_TOOLONG', 61, 'Field is too long for this implementation')
+            KDC_ERR_CLIENT_NOT_TRUSTED = ErrorCode.new('KDC_ERR_CLIENT_NOT_TRUSTED', 62, 'PKINIT - KDC_ERR_CLIENT_NOT_TRUSTED')
+            KDC_ERR_INVALID_SIG = ErrorCode.new('KDC_ERR_INVALID_SIG', 64, 'PKINIT - KDC_ERR_INVALID_SIG')
+            KDC_ERR_DH_KEY_PARAMETERS_NOT_ACCEPTED = ErrorCode.new('KDC_ERR_DH_KEY_PARAMETERS_NOT_ACCEPTED', 65, 'PKINIT - KDC_ERR_DH_KEY_PARAMETERS_NOT_ACCEPTED')
```

Comment:
Microsoft differs from the pkinit spec, refers to this as `KDC_ERR_KEY_TOO_WEAK`: https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4768#table-2-kerberos-ticket-flags

I'm impartial on which naming convention to use unless someone has stronger opinions here

Reply:
I think following the PKINIT spec makes the most sense since long-term we're looking to support more than just Microsoft's Kerberos implementation.

Reply:
Looks like the older Kerberos spec used `KDC_ERR_KEY_TOO_WEAK` here. Not a blocker for me either way :+1: